README.md

```diff
@@ -73,47 +75,40 @@ macOS tools:
 
 Unix tools:
 
-* [Git] for version control
 * [htop] for viewing process information in the CLI
 * [OpenSSL] for Transport Layer Security (TLS)
 * [The Silver Searcher] for finding things in files
-* [vim] as a handy in-terminal editor for rainy days
+* [tmux] for saving project state and switching between projects
```

Comment:
thoughtbot added this upstream and I agree.

Reply:
Tmux4lyfe!